pkg/smartcontract/manifest/standard/nep11.go

```diff
@@ -0,0 +1,152 @@
+package standard
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+)
+
+var nep11Base = &Standard{
+	Manifest: manifest.Manifest{
+		ABI: manifest.ABI{
+			Methods: []manifest.Method{
+				{
+					Name: "balanceOf",
+					Parameters: []manifest.Parameter{
+						{Type: smartcontract.Hash160Type},
+					},
+					ReturnType: smartcontract.IntegerType,
+					Safe:       true,
+				},
+				{
+					Name:       "decimals",
+					ReturnType: smartcontract.IntegerType,
+					Safe:       true,
+				},
+				{
+					Name:       "properties",
+					ReturnType: smartcontract.MapType,
+					Parameters: []manifest.Parameter{
+						{Type: smartcontract.ByteArrayType},
+					},
+					Safe: true,
+				},
+				{
+					Name:       "symbol",
+					ReturnType: smartcontract.StringType,
+					Safe:       true,
+				},
+				{
+					Name:       "tokensOf",
+					ReturnType: smartcontract.AnyType, // Iterator
```

Comment:
`InteropInterfaceType`?

Reply:
At the current moment in C# node native NameService has `AnyType` here (that was one of the reasons for our state diff). Although the standard assumes `InteropInterface`. I think that we may create an issue in C# node repo.